<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsbPrintWindow</class>
 <widget class="QMainWindow" name="UsbPrintWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 226, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>20</y>
      <width>401</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>14</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 120%;
text-align: center;</string>
     </property>
     <property name="text">
      <string>ПЕЧАТЬ ИЗ USB</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>30</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>501</width>
      <height>370</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EDF3F6;
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>91</y>
      <width>232</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FDFEFF;
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="modelNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 16px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #EDF3F6;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Время печати:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Вес модели:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Слоёв:</string>
     </property>
    </widget>
    <widget class="QLabel" name="layersLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>186</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>227</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="weightLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>266</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 14px;
line-height: 130%;
color: #12222D;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>403</y>
      <width>232</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/start_print.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="connectionWifi">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/noSignal.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
